src/microbe_stage/MicrobeStage.cs

```diff
@@ -518,17 +526,19 @@ public override void OnReturnFromEditor()
         {
             daughter.BecomeFullyGrownMulticellularColony();
 
-            // TODO: add more extra offset between the player and the divided cell
-            // See: https://github.com/Revolutionary-Games/Thrive/issues/3653
-        }
+            if (daughter.Colony != null)
+            {
+                // Add more extra offset between the player and the divided cell
+                var daughterPosition = daughter.GlobalTransform.origin;
+                var direction = (playerPosition - daughterPosition).Normalized();
 
-        // Update the player's cell
-        Player.ApplySpecies(Player.Species);
+                var colonyMembers = daughter.Colony.ColonyMembers.Select(c => c.GlobalTransform.origin).ToList();
```

Comment:
The ToList call can be saved here as well if the change to GetMaximumDistanceInDirection is done.